django/core/serializers/base.py

```diff
@@ -125,8 +125,12 @@ def serialize(
         self.selected_fields = fields
         self.use_natural_foreign_keys = use_natural_foreign_keys
         self.use_natural_primary_keys = use_natural_primary_keys
+        self.extra_fields = options.pop("extra", None)
         progress_bar = self.progress_class(progress_output, object_count)
 
+        def field_is_selected(field):
```

Comment:
Nitpicky: but I find it more readable when the name starts with the verb when the function returns a boolean.
```suggestion
        def is_field_selected(field):
```